[DOCS-2830] Update support ticket page (#12527)
DataDog/documentation: content/en/account_management/faq/access-your-support-ticket.md

## Creating a ticket

{{< site-region region="us,us3,us5,eu" >}}
To create a new support ticket, go to [https://help.datadoghq.com][1], and click **Submit a request** to fill out a ticket form.

[1]: https://help.datadoghq.com
{{< /site-region >}}
{{< site-region region="gov" >}}
To create a new support ticket, go to [https://help.dd-gov.com][1], and click **Submit a request** to fill out a ticket form.

[1]: https://help.ddog-gov.com
{{< /site-region >}}

## Accessing existing tickets

If you have opened at least one Datadog support ticket, follow this process to access all your Datadog support tickets:

{{< site-region region="us,us3,us5,eu" >}}
1. Go to [https://help.datadoghq.com][1] and click **Sign in** on the top right. 

> 2. If this is your first time signing into Zendesk, click **New to your Datadog Zendesk account? Sign up** to create a Zendesk account. If you have previously emailed Datadog Support, click **Have you emailed us? Get a password**.

Reviewer: ```suggestion
2. If this is your first time signing into Zendesk, click **New to your Datadog Zendesk account? Sign up** to create a Zendesk account. If you have previously emailed Datadog support, click **Have you emailed us? Get a password**.
```

>     **Note**: The email you use must match the email you used to contact Datadog Support.

Reviewer: ```suggestion
    **Note**: You must enter the email address you used to contact Datadog support.
```

  {{< img src="help/sign_in.png" alt="Sign into Zendesk" style="width:70%" >}}

[1]: https://help.datadoghq.com
{{< /site-region >}}
{{< site-region region="gov" >}}
1. Go to [https://help.dd-gov.com][1] and click **Sign in** on the top right. 

> 2. If this is your first time signing into Zendesk, click **New to Datadog? Sign up** to create a Zendesk account. If you have previously emailed Datadog Support, click **Have you emailed us? Get a password**.

Reviewer: ```suggestion
2. If this is your first time signing into Zendesk, click **New to Datadog? Sign up** to create a Zendesk account. If you have previously emailed Datadog support, click **Have you emailed us? Get a password**.
```

>     **Note**: The email you use must match the email you used to contact Datadog Support.

Reviewer: ```suggestion
    **Note**: You must enter the email address you used to contact Datadog support.
```

  {{< img src="help/sign_in_gov.png" alt="Sign into Zendesk" style="width:50%" >}}

[1]: https://help.ddog-gov.com
{{< /site-region >}}

3. After you receive the password in your email, log in and click **Manage your tickets** to see your requests:
  {{< img src="help/my_activities.png" alt="my activities"  >}}

> 5. If you don't see the **My Activities** page after logging in, click on your name in the upper right hand corner, and then click **My Activities**.

Reviewer: ```suggestion
5. If you don't see the **My Activities** page after logging in, click on your name in the upper right corner, and then click **My Activities**.
```
